internal/cmd/commands/authenticate/authenticate.go

```diff
@@ -35,5 +48,42 @@ func (c *Command) Help() string {
 }
 
 func (c *Command) Run(args []string) int {
-	return cli.RunResultHelp
+	client, err := c.Client(base.WithNoTokenScope(), base.WithNoTokenValue())
+	if c.WrapperCleanupFunc != nil {
+		defer func() {
+			if err := c.WrapperCleanupFunc(); err != nil {
+				c.PrintCliError(fmt.Errorf("Error cleaning kms wrapper: %w", err))
+			}
+		}()
+	}
+	if err != nil {
+		c.PrintCliError(fmt.Errorf("Error creating API client: %w", err))
+		return base.CommandCliError
+	}
+
+	// Lookup the primary auth method ID in the global scope
+	aClient := authmethods.NewClient(client)
+	pri, err := getPrimaryAuthMethodId(c.Context, aClient, scope.Global.String(), "")
```

Comment:
Can we allow a `-scope-id` flag here to override this if the auth method ID isn't known but scope ID is?

Reply:
We decided to restrict the flag scope to a sub command due to limitations in the flag library: the -scope-id flag would move up in precedence and that would modify the CLI usage, and CLI usage changes of existing commands was out of scope for this PR. To maintain the existing CLI usage and have the flag remain the precedence in the command it is for password and OIDC, we could not use it at both the `boundary authenticate` level and the `boundary authenticate <auth method>` level. Fun times.